applications/serial_lte_modem/src/slm_at_tcp_proxy.c

```diff
@@ -448,27 +451,27 @@ static void tcpsvr_thread_func(void *p1, void *p2, void *p3)
 		if (ret == 0) { /* timeout */
 			continue;
 		}
-		LOG_DBG("fds[0] events 0x%08x", fds[0].revents);
-		LOG_DBG("fds[1] events 0x%08x", fds[1].revents);
+		LOG_DBG("listen events 0x%08x", fds[SOCK_LISTEN].revents);
+		LOG_DBG("peer events 0x%08x", fds[SOCK_PEER].revents);
+		LOG_DBG("efd events 0x%08x", fds[EVENT_FD].revents);
 		/* Listening socket events must be handled first*/
-		if (fds[0].revents) {
-			if ((fds[0].revents & POLLERR) == POLLERR) {
-				LOG_ERR("0: POLLERR");
+		if (fds[SOCK_LISTEN].revents) {
+			if ((fds[SOCK_LISTEN].revents & POLLERR) != 0) {
+				LOG_WRN("%d: POLLERR", fds[SOCK_LISTEN].fd);
```

Comment:
Outputting the actual fd integer is not informative (as not predictable or requiring investigation to figure it out), is it? Better change to e.g. `SOCK_LISTEN` only.

Reply:
I think that the fd would be beneficial as well. Can be both.

Reply:
Okay for the FD to be there if you think it's useful, as long as there is at least one stable, easily recognizable identifier.